.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -29,17 +29,32 @@ body:
     validations:
       required: true
   - type: textarea
-    id: environment
+    id: os-and-postgresql-version
     attributes:
-      label: Environment
-      description: Tell us your environment.
+      label: OS and PostgreSQL version
+      description: Tell us your OS version and PostgreSQL version.
       value: |
         - OS: [e.g. Ubuntu]
         - OS Version: [e.g. Ubuntu 24.04.2 LTS]
         - PostgreSQL Version: [e.g. 15.1]
-        - PGroonga Version: [e.g. 12.1.0]
-        - libgroonga Version: [e.g.  8.0.1]
-           - https://pgroonga.github.io/reference/parameters/libgroonga-version.html
+    validations:
+      required: true
```

Comment:
Can we remove this?
We have some environment free issues such as https://github.com/pgroonga/pgroonga/issues/708 .

Reply:
Ah, we can use "feature request" issue for it?

Reply:
Yes, I think we should use "feature request" issue for some environment free issues.